prelude.dx

```diff
@@ -91,6 +91,11 @@ def fst (p: (a & b)) : a = %fst p
 def snd (p: (a & b)) : b = %snd p
 def swap (p:(a&b)) : (b&a) = (snd p, fst p)
 
+def astype (t:Type) (x:t) : t = x  -- to give hints to type inference
```

Comment:
As above, I'm a bit reluctant to add this to prelude because it serves exactly the same purpose as the type annotations that we have built into the syntax.

Reply:
(see above)